src/epgsql_sock.erl

```diff
@@ -89,13 +91,20 @@
                 repl_feedback_required,
                 repl_cbmodule,
                 repl_cbstate,
-                repl_receiver}).
+                repl_receiver,
+                timer}).
 
 %% -- client interface --
 
 start_link() ->
     gen_server:start_link(?MODULE, [], []).
 
+command(C, Command) ->
+  gen_server:call(C, Command).
```

Comment:
Ah, and default `Timeout` for `gen_server:call` is 5s. We should pass `infinity` here.